modular_skyrat/modules/death_consequences_perk/death_consequences.dm

```diff
@@ -0,0 +1,73 @@
+/datum/quirk/death_consequences
+	name = DEATH_CONSEQUENCES_QUIRK_NAME
+	desc = "Every time you die, your body suffers long-term damage that can't easily be repaired. This quirk is highly customizable - See 'DC Config' \
+	in bottom-right Character preferences section after selecting it."
+	medical_record_text = DEATH_CONSEQUENCES_QUIRK_DESC
+	icon = FA_ICON_DNA
+	value = 0 // due to its high customization, you can make it realllly good or reallllly bad
+
+/datum/quirk/death_consequences/add(client/client_source)
+	var/mob/living/carbon/human/human_holder = quirk_holder
+	human_holder.gain_trauma(/datum/brain_trauma/severe/death_consequences, TRAUMA_RESILIENCE_ABSOLUTE)
+
+	add_verb(human_holder, TYPE_VERB_REF(/mob, adjust_degradation))
+	add_verb(human_holder, TYPE_VERB_REF(/mob, refresh_death_consequences))
+
+	to_chat(human_holder, span_danger("You suffer from [DEATH_CONSEQUENCES_QUIRK_NAME]. By default, you will \
+	degrade every time you die, and recover very slowly while alive. This may be expedited by resting, sleeping, being buckled \
+	to something cozy, or using rezadone. \n\
+	As your degradation rises, so too will negative effects, such as stamina damage or a worsened crit threshold. \n\
+	You can alter your degradation on the fly via the Adjust resonance degradation verb, and change your settings via the Refresh death consequence variables verb."))
```

Comment:
```suggestion
		degrade every time you die, and recover very slowly while alive. This may be expedited by resting, sleeping, being buckled \
		to something cozy, or using rezadone.\n\
		As your degradation rises, so too will negative effects, such as stamina damage or a worsened crit threshold.\n\
		You can alter your degradation on the fly via the Adjust resonance degradation verb, and change your settings via the Refresh death consequence variables verb."))
```